techniques/system/server-roles/1.0/relayd.conf.tpl

```diff
@@ -73,7 +73,7 @@ command = "/opt/rudder/bin/rudder"
 use_sudo = true
 
 [shared_files]
-path = "{{{vars.system_common.sharedfiles_folder}}}"
```

Comment:
this one actually had the same value as `vars.g.shared_files`. We are paying our poor directory naming I guess.